src/pages/Main/NgoSignup.js

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect } from "react";
+import { NgoForm } from "@components/VolunteerForm";
+import { connecter } from "@store/ngoSignup";
+import options from "@utils/Options";
+import formatter from "@utils/Formatter";
+
+function NgoSignup({ reset, mode, save }) {
```

Comment:
mode may not be needed here, it's used in forms which have a toggle for individual/organisation or some form type.